### DISCUSSION TITLE
profile loading time fixed!1!11!!!11!111!!!

### DISCUSSION THREAD

**Discussion by davwat** [CluckCluckChicken/Magnifier#89] [Announcements]
epic gamer moment

look at this wonderful screenshot:
![magnifier request](https://user-images.githubusercontent.com/46281840/120834075-770cfb00-c55a-11eb-99fb-67140116b0cb.png)

**Comment by JackyBorderCollie** [comment]
yeeeeeeeeeeeeeeeeeeees

**Comment by JackyBorderCollie** [comment]
add epik gamer label
